src/web/src/components/SignUp/Forms/RSSFeeds.tsx

```diff
@@ -0,0 +1,264 @@
+      margin: '0',
+      position: 'relative',
+      width: '100%',
+      minHeight: '100%',
+    },
+    container: {
+      display: 'grid',
+      gridTemplateAreas: '1fr',
+      textAlign: 'center',
+      justifyItems: 'center',
+      alignItems: 'center',
+      position: 'absolute',
+      minHeight: '100%',
+      width: '100%',
+      [theme.breakpoints.down(600)]: {
+        width: '95%',
+        marginLeft: '2.5%',
+      },
+    },
+    blogPageTitle: {
+      fontSize: '1.5em',
+    },
+    helpText: {
+      fontSize: '1.1em',
+      lineHeight: '1.8em',
+    },
+    infoContainer: {
+      display: 'grid',
+      textAlign: 'center',
+      gridGap: '10%',
+      justifyItems: 'center',
+      alignItems: 'center',
+      width: '90%',
+    },
+    inputsContainer: {
+      width: '100%',
+      display: 'grid',
+      gridTemplateColumns: '75% 25%',
+      '& .MuiFormHelperText-root': {
+        fontSize: '0.9em',
+        color: 'black',
+      },
+      '& .MuiFormLabel-root': {
+        color: 'black',
+      },
+      [theme.breakpoints.down(600)]: {
+        gridTemplateColumns: '80% 20%',
+      },
+    },
+    helpMessage: {
+      fontSize: '.9em',
+      color: 'black',
+    },
+    button: {
+      height: '35px',
+      width: '50%',
+      alignSelf: 'center',
+      fontSize: '0.8em',
+      marginLeft: '5%',
+      background: '#121D59',
+      color: '#A0D1FB',
+      '&:hover': {
+        color: 'black',
+        border: '1px solid #121D59',
+      },
+      '&.Mui-disabled': {
+        backgroundColor: 'inherit',
+      },
+    },
+    RssButtonContainer: {
+      width: '90%',
+      display: 'grid',
+    },
+    infoRSSContainer: {
+      minHeight: '120px',
+      maxHeight: '120px',
+      width: '100%',
+      overflowY: 'auto',
+    },
+    noBlogMessage: {
+      fontSize: '1em',
+      color: '#474747',
+      marginTop: '40px',
+    },
+    text: {
+      fontSize: '0.9em',
+      alignSelf: 'end',
+      color: '#474747',
+    },
+    RssButtonWrapper: {
+      width: '100%',
+    },
+    RssButton: {
+      width: '101%',
+      borderRadius: '0',
+      background: '#121D59',
+      color: '#A0D1FB',
+      '&:hover': {
+        color: 'black',
+        border: '1px solid #121D59',
+      },
+    },
+    agreeMessage: {
+      [theme.breakpoints.down(600)]: {
+        alignSelf: 'end',
+      },
+    },
+    formControlLabel: {
+      fontSize: '.9em',
+      height: '10px',
+      color: '#474747',
+    },
+  })
+);
+
+const RSSFeeds = connect<{}, SignUpForm>((props) => {
+  const classes = useStyles();
+  const { values, errors, setFieldValue } = props.formik;
+  const { token } = useAuth();
+
+  const [feedUrls, setFeedUrls] = useState<Array<string>>([]);
+  const [blogUrlError, setBlogUrlError] = useState('');
+  const [validating, setValidating] = useState(false);
+
+  // A controller to cancel undesired requests
+  const [controller, setController] = useState(new AbortController());
```

Comment:
I think this makes more sense as a Ref vs. State, https://frontend-digest.com/cancelling-fetch-requests-in-react-applications-58a52a048e8e

Reply:
@DukeManh 
I think this is done right?